src/types/response.ts

```diff
@@ -21,7 +21,8 @@ import { Erc20Metadata } from './erc20Metadata'
 export type AdapterErrorResponse = {
   error: {
     message: string
-    details?: Record<string, unknown>
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    details?: any
```

Comment:
I've done this because in both cases we don't know what this is going to have, but with `any` we can at least check it without typescript complaining.